extract_data.py

```diff
@@ -97,31 +96,85 @@ def extract_and_collect_data_from_generated_files(directory: str, has_dedicated_
             "frequency-hertz": None,
             "human_readable_frequency": None
         }
-
-    result = [chassis, mobo, cpu]
-    if wifi_cards:
-        result += wifi_cards
-
-    if isinstance(dimms, dict):
-        # otherwise it will append every key-value pair of the dict
-        result.append(dimms)
-    else:
+    """
+    def normalize_brands(coll_dict):
+        names = {}
+        with open("normalized.csv", "r") as f:
+            while True:
+                line = f.readline()
+                if line == "":
+                    break
+                k, v = line.split(";", 2)[0:2]
+                names[k] = v
+        for p_dict in coll_dict:
+            if "brand" in p_dict.keys():
+                if p_dict["brand"] in names.keys():
+                    p_dict["brand"] = names[p_dict["brand"]]
+
+    item_keys = ["arrival-batch", "cib", "cib-old", "cib-qr", "data-erased", "mac", "notes",
+                      "os-license-code", "os-license-version", "other-code", "owner", "smart-data",
+                      "sn", "software", "surface-scan", "working", "wwn"]
+    bmv = ["brand", "model", "variant"]
+
+    normalize_brands([chassis, mobo, cpu, gpu, psu]+dimms+wifi_cards+disks)
```

Comment:
```suggestion
    normalize_brands([chassis, mobo, cpu, gpu, psu] + dimms + wifi_cards + disks)
```
Also, are these (chassis, mobo, cpu, gpu, psu) always not lists and these (dimms + wifi_cards + disks) always lists or is a check needed? If needed, you can use `if instanceof(stuff, list)`.  
Please fix the error thrown at this line, here's a stacktrace:
```python
Traceback (most recent call last):
  File "./extract_data.py", line 352, in <module>
    cleanup=True)
  File "./extract_data.py", line 119, in extract_and_collect_data_from_generated_files
    normalize_brands([chassis, mobo, cpu, gpu, psu]+dimms+wifi_cards+disks)
TypeError: can only concatenate list (not "NoneType") to list

```